PLUGIN_MIGRATION.md

```diff
@@ -69,3 +69,7 @@ Optionally change the `routes()` to `routeHandlers()`.  Change `prepareRequest()
 ### Replace BytesRestResponse with ExtensionRestResponse
 
  - Add the `request` as the first parameter, the remainder of the parameters should be the same.
+
+### Replace Return Type for SDKRestClient
+
+- Replace the return type to `ParsedStringTerms` from `StringTerms` to fetch the aggregation for a specific index.
```

Comment:
Can we introduce this with a bit more of a generic paragraph? Something like:

While most SDKRestClient client return types match existing classes, some changes may be necessary to conform to the new method signatures.  Examples include:

Another example is this diff in AnomalyDetectionIndices:
```diff
-                for (ObjectObjectCursor<String, List<AliasMetadata>> entry : getAliasResponse.getAliases()) {
-                    if (false == entry.value.isEmpty()) {
+                for (Entry<String, Set<AliasMetadata>> entry : getAliasResponse.getAliases().entrySet()) {
+                    if (false == entry.getValue().isEmpty()) {
```

Reply:
Let me know what do you think of the new change.